package.json

```diff
@@ -286,7 +286,7 @@
         "react-dom": "^16.9.0",
         "react-draggable": "^3.3.0",
         "react-immutable-proptypes": "^2.1.0",
-        "react-intl": "^2.8.0",
+        "react-intl": "2.9.0",
```

Comment:
Peer dependency upgrades should be marked as a breaking change via the commit message.